src/EvoSC.Common/Config/Models/IDatabaseConfig.cs

```diff
@@ -6,6 +6,9 @@ namespace EvoSC.Common.Config.Models;
 
 public interface IDatabaseConfig
 {
+    /// <summary>
+    /// Database types supported by the application.
```

Comment:
Database types supported by the application e.g MySQL, PostgreSQL.

Reply:
only adding mysql as its the only one supported atm